src/lib/openapi/spec/project-flag-creators-schema.ts

```diff
@@ -0,0 +1,32 @@
+import type { FromSchema } from 'json-schema-to-ts';
+
+export const projectFlagCreatorsSchema = {
+    $id: '#/components/schemas/projectFlagCreatorsSchema',
+    type: 'array',
+    description: 'A list of project flag creators',
+    items: {
+        type: 'object',
+        additionalProperties: false,
+        required: ['id', 'name'],
+        properties: {
+            id: {
+                type: 'integer',
+                example: 50,
+                description: 'The user id.',
+            },
+            name: {
+                description: 'Name of the user',
```

Comment:
Can we mention how we use fallbacks here if there is no real name?

```suggestion
                description: 'Name of the user. If the user has no set name, the API falls back to using the user's username (if they have one) or email (if neither name or username is set).',
```